docs/integrations/r.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: default
+title: Using R with lakeFS
+description: How to use lakeFS from R including creating branches, committing changes, and merging.
+parent: Integrations
+nav_order: 60
+has_children: false
+---
+
+# Using R with lakeFS
+
+R is a powerful language used widely in data science. lakeFS interfaces with R in two ways: 
+
+*  To **read and write data in lakeFS** use standard S3 tools such as the `aws.s3` library. lakeFS has a [S3 gateway](https://docs.lakefs.io/understand/architecture.html#s3-gateway) which presents a lakeFS repository as an S3 bucket.
```

Comment:
Don't know where it should be specified - the fact that s3 gateway represents a branch as a folder.

Reply:
Added